src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_SIZE = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    private int getRandomSize() {
+        return random.nextInt(10);

```

Comment:
```suggestion
        return random.nextInt(DEFAULT_SIZE);
```
Move private method after public